test/test_tools_add_pre_post_processing_to_model.py

```diff
@@ -170,8 +181,144 @@ def test_pytorch_superresolution(self):
         # whether avx512 is used or not. MacOS seems to be slightly worse though (max of 2)
         diffs = np.absolute(expected.astype(np.int32) - result.astype(np.int32))
         total = np.sum(diffs)
-        print(f'Max diff:{diffs.max()} Total diffs:{total}')
+        print(f"Max diff:{diffs.max()} Total diffs:{total}")
         self.assertTrue(diffs.max() < 3 and total < (result.size / 1000))
 
+    def build_testModel_and_get_ref_output_for_tokenizer(
+        self, tokenizer_type: str, output_model: Path, pre_steps: List[pre_post_processing.steps.Step] = []
+    ):
+        # import transformers
+        import onnx
+
+        create_named_value = pre_post_processing.utils.create_named_value
+        SentencePieceTokenizer = pre_post_processing.steps.SentencePieceTokenizer
+        TokenizerParam = pre_post_processing.steps.TokenizerParam
+        BertTokenizer = pre_post_processing.steps.BertTokenizer
```

Comment:
Given it's test code could we just import * from pre_post_processing.steps and not need these lines of code that seems to just be about removing the 'pre_post_processing.steps.' prefix.